transport/handler.go

```diff
@@ -37,3 +68,17 @@ type Handler interface {
 		resw ResponseWriter,
 	) error
 }
+
+// OnewayHandler handles a single, transport-level, oneway request.
+type OnewayHandler interface {
+	// Handle the given oneway request
+	//
+	// An error may be returned in case of failures. BadRequestError must be
```

Comment:
whoops, pasted from the comments from Handler interface, haven't fully thought out what errors here should look like yet